### PR TITLE
feat: add example1

### PR DESCRIPTION
**Pull request by FacerAin**
## Description
* Add example1 for testing

## Related Issue
* No Related Issues

### CODE REVIEW

File: example/example1.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("Example 1 code")
```
Comment (jinmyeongAN) [inline example/example1.py]: 시간복잡도 측면에서 좋아보입니다 :)

**Comment by FacerAin** [comment]
#1

File: example/example1.py (hunk 1 above)
Comment (FacerAin) [inline example/example1.py]: review 1

Comment (kyc3492) [reply example/example1.py]: LGTM 👍

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
-repos:
--   repo: local
-    hooks:
-    - id: style
-      name: style
-      language: system
-      entry: make style
-      types: [python]
+repos:
```
Comment (wbin0718) [inline .pre-commit-config.yaml:1]: good!

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+-   repo: local
+    hooks:
+    - id: style
```
Comment (ghlrobin) [inline .pre-commit-config.yaml:4]: i don't think so...

**Review by jinmyeongAN — APPROVED** [review]
So much good!

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+      name: style
+      language: system
+      entry: make style
```
Comment (ghlrobin) [inline .pre-commit-config.yaml:7]: this is unnecessary

**Review by ghlrobin — CHANGES_REQUESTED** [review]
Good job  overall, but check my review

**Review by wbin0718 — APPROVED** [review]

**Review by ghlrobin — APPROVED** [review]

**Review by kyc3492 — APPROVED** [review]
Goooooooood

**Pull request merged**